### ISSUE TITLE
Uncaught exceptions not caught, when using Webpack

### ISSUE THREAD

**Issue by JuanCaicedo** [rollbar/rollbar.js#677]
Hi there! I can flush this out with some more details and example code if it would help 😄

I was trying to get Rollbar building through webpack. I found that I could get it working, so that if from another file in that build I call Rollbar, then it works
```js
Rollbar.error('Manually calling Rollbar.error');
```

However, right below it I tried an uncaught exception. This exception was not caught by Rollbar
```js
throw new Error('from app.js in build');
```

I get the same result if I run this example code from the docs
```js
window.onerror("TestRollbarError: testing window.onerror", window.location.href)
```

I instantiated Rollbar with both of these settings
```js
       captureUncaught: true,
       captureUnhandledRejections: true,
```

However, if I load Rollbar through the snippet method instead of webpack, all errors are sent to Rollbar.com without any problems.

Again, I can provide more info if it's helpful, but seeing other issues like https://github.com/rollbar/rollbar.js/issues/567, I got to wondering, is building through Webpack supported?

**Comment by ajtran** [comment]
@jessewgibbs I've been in contact with the user via intercom - https://app.intercom.io/a/apps/rlisg82q/inbox/inbox/488687/conversations/18734322864

I was hoping to get more eyes on building via webpack.

**Comment by jessewgibbs** [comment]
@rokob do you mind taking a look at this?

**Comment by jessewgibbs** [comment]
@juancaicedo sorry it's taken us a while to get to this.  Are you still encountering this problem?

**Comment by agriffis** [comment]
I'm seeing this problem. Just caused us a big headache, actually, as an app was failing and we weren't getting errors in rollbar

**Comment by jessewgibbs** [comment]
@agriffis thanks for the quick reply and sorry for the hassle!

@rokob can you investigate this?

**Comment by JuanCaicedo** [comment]
I've moved on to work on something else, but hopefully @agriffis can elaborate more 👍

**Comment by agriffis** [comment]
Actually now that I've taken a closer look, I think it's working fine for me.  In my case, I was missing the setup for [Vue render exceptions](https://rollbar.com/blog/error-tracking-vue-js/):

```
Vue.config.errorHandler = err => {
    Rollbar.error(err)
}
```

I was also misled by attempting to test in the Chrome dev tools. I created a function in my source code:

```
window.testError = () => {
    throw new Error('testing')
}
```

When I called that from the Chrome dev tools, it wasn't caught by `window.onerror`. That seems to be a Chrome dev tools feature... The error isn't raised in the window context, so it isn't caught by the error handler.

However this demonstrated that it worked fine:

```
window.testError = () => {
    throw new Error('testing')
}

setTimeout(window.testError, 2000)
```

In summary:
* `captureUncaught` seems to be working fine for me in a Webpack app. The problem was that the framework (Vue) was also catching exceptions and needed to be configured to use Rollbar.
* Throwing an error from the debug console doesn't exercise the `window.onerror` handler.

**Comment by jessewgibbs** [comment]
@agriffis thanks for the detailed follow-up.  I'm gonna go ahead and close this issue now but please let us know if you encounter any further issues.
